src/Entities/Donation.php

```diff
@@ -221,6 +221,15 @@ class Donation {
 	 */
 	private $addressChange;
 
+	/**
+	 * Donation constructor creates a new AddressChange entity unless one is supplied by Doctrine
```

Comment:
I think this info is not needed by whoever constructs the object and is thus better left out. Perhaps something can be added to getAddressChange. I don't think that is needed, but that location would make more sense to me.